src/main/java/core/basesyntax/Lottery.java

```diff
@@ -3,9 +3,11 @@
 import java.util.Random;
 
 public class Lottery {
-    private static final int maxNumber = 100;
+    private static final int RANDOM_RANGE_LIMIT = 100;
+    private static final ColorSupplier COLOR_SUPPLIER = new ColorSupplier();
```

Comment:
It is better if `ColorSupplier` be non-static variable, it is just usual property. The same for the `Random` property. Do not forget to change the uppercase letters when renaming.